src/components/Editor/Tab.js

```diff
@@ -135,13 +137,24 @@ class Tab extends PureComponent<Props> {
       }
     ];
 
-    items.push({
-      item: {
-        ...tabMenuItems.prettyPrint,
-        click: () => togglePrettyPrint(tab),
-        disabled: !shouldShowPrettyPrint(source)
+    items.push(
```

Comment:
I think we want to add `shouldBlackbox` here since we use it for other places.

Reply:
Added the changes!